src/agile_object.rs

```diff
@@ -22,7 +22,7 @@ unsafe impl ComInterface for IAgileObject {
 #[repr(C)]
 pub struct abi_IAgileObject {
     pub unknown_query_interface:
-        extern "system" fn(RawComPtr<IUnknown>, &Guid, *mut RawPtr) -> ErrorCode,
-    pub unknown_add_ref: extern "system" fn(RawComPtr<IUnknown>) -> u32,
-    pub unknown_release: extern "system" fn(RawComPtr<IUnknown>) -> u32,
+        unsafe extern "system" fn(RawComPtr<IUnknown>, &Guid, *mut RawPtr) -> ErrorCode,
```

Comment:
This is a preview of a change to come. All abi methods should be marked unsafe since it's up to us to make sure they're used properly.